magento/magento2
# Remove casting to int on increment ID which can be prefixed by a string

### Description (*)
- Remove casting to int on increment ID which can be prefixed by a string

### Manual testing scenarios (*)
1. Configure a string prefix to invoice increment IDs
2. Submit an order
3. Create an invoice
4. Go to customer account -> orders -> invoice -> print invoice
5. The increment ID should be there, but is "0" for numbers prefixed with a string

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#34451: Remove casting to int on increment ID which can be prefixed by a string
